.changeset/soft-crabs-flow.md

```diff
@@ -0,0 +1,19 @@
+---
+'@itwin/itwinui-react': major
+---
+
+Added new `Breadcrumbs.Item` subcomponent to use for custom `a` and `button` elements within the `Breadcrumbs`
+
+```jsx
```

Comment:
Remove example, simply list components that are available. We have documentation site for examples.